.github/workflows/publish.yml

```diff
@@ -26,6 +26,7 @@ jobs:
       # - name: testing
       #   run: npm run test
       - name: Publish package on NPM 📦
-        run: npx pnpm --filter "@klaytn-developer-sdk/*" -r publish --publish-branch main --no-git-check
-        env:
-          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+        uses: JS-DevTools/npm-publish@v1
```

Comment:
Hi @Sotatek-TinnNguyen , Publish should perform below things.
1) It should publish only the sub packages (kds-cli, oracle, celer, wormhole)  and avoid publishing the root package.
2) It should only publish the sub package which package.json's version got changed. Ex: if only oracle subpackage version is changed, on that needs be published.

May i know if this performs as mentioned above ?

Reply:
So when we run the publish.yml below packages should get published. It should avoid publishing root package.
@klaytn-developer-sdk/kds-cli   -> path=/packages/kds-cli
@klaytn-developer-sdk/oracles   -> path=/packages/oracles-starter-kit
@klaytn-developer-sdk/bridges-wormhole   -> path=/packages/bridges-starter-kit/wormhole
@klaytn-developer-sdk/bridges-celer   -> path=/packages/bridges-starter-kit/celer
@klaytn-developer-sdk/dexs   -> path=/packages/dexs-starter-kit